src/app/cli/src/commands/sql_shell_command.rs

```diff
@@ -42,9 +44,13 @@ pub struct SqlShellCommand {
     engine_prov_config: Arc<EngineProvisionerLocalConfig>,
     output_config: Arc<OutputConfig>,
     container_runtime: Arc<ContainerRuntime>,
+    export_service: Arc<dyn ExportService>,
     command: Option<String>,
     url: Option<String>,
     engine: Option<SqlShellEngine>,
+    output_path: Option<PathBuf>,
+    partition_size: Option<usize>,
+    supported_export_formats: Vec<OutputFormat>,
```

Comment:
I guess you can declare it as `const` in the module. Does not feel like it's a state.